tools/aev-benchmark-size.py

```diff
@@ -26,22 +26,41 @@ def info(text):
     print('\033[32m{}\33[0m'.format(text))  # green
 
 
-def benchmark(speciesPositions, aev_comp, N, check_gpu_mem):
+def benchmark(speciesPositions, aev_comp, N, check_gpu_mem, nn=None, verbose=True):
     torch.cuda.empty_cache()
     gc.collect()
     torch.cuda.synchronize()
     start = time.time()
 
     aev = None
     for i in range(N):
-        aev = aev_comp(speciesPositions).aevs
+        species, coordinates = speciesPositions
+        if nn is not None:  # double backward
+            # graph1 input -> aev
+            coordinates = coordinates.requires_grad_()
+            _, aev = aev_comp((species, coordinates))
+            # graph2 aev -> E
+            aev_ = aev.clone().detach().requires_grad_()
```

Comment:
Why do you need to detach in benchmark? Real force training does not detach, and benchmark should be consistent with real applications.